ci/README.md

```diff
@@ -0,0 +1,258 @@
+# Configuring the CI Environment
+
+We use several Google Cloud Platform services to perform integration tests.
+These notes describe how to setup a project to run these integration tests.
+The main audience for these notes are developers working to improve the
+Functions Framework for C++. They assume the reader is familiar with GCP and
+with the Google Cloud SDK command-line tool.
+
+## Pre-requisites
+
+These notes assume you have setup a valid GCP project, with billing enabled.
+
+> :warning: Most of these services will incur billing costs, probably in the
+> hundreds of dollars per month. Do not run these steps unless you understand
+> these costs.
+
+In these notes we will use `GOOGLE_CLOUD_PROJECT=...` as a shell variable with
+the name of the project you want to use.
+
+## Enable the Services
+
+```sh
+gcloud services enable --project="${GOOGLE_CLOUD_PROJECT}" \
+    bigtable.googleapis.com \
+    cloudbuild.googleapis.com \
+    cloudfunctions.googleapis.com \
+    container.googleapis.com \
+    containerregistry.googleapis.com \
+    eventarc.googleapis.com \
+    logging.googleapis.com \
+    pubsub.googleapis.com \
+    run.googleapis.com \
+    spanner.googleapis.com
+```
+
+## Create Pub/Sub Topics
+
+```sh
+gcloud pubsub topics create testing \
+  --project="${GOOGLE_CLOUD_PROJECT}"
+gcloud pubsub topics create gcs-changes \
+  --project="${GOOGLE_CLOUD_PROJECT}"
+```
+
+## Create a Bucket
+
+```sh
+gsutil mb -p "${GOOGLE_CLOUD_PROJECT}" \
+    "gs://${GOOGLE_CLOUD_PROJECT}-testing-bucket"
+```
+
+### Grant the GCS service permissions to publish to Pub/Sub
+
+Get the GCS service account:
+
+```shell
+curl -X GET -H "Authorization: Bearer $(gcloud auth print-access-token)" \
+  "https://storage.googleapis.com/storage/v1/projects/${GOOGLE_CLOUD_PROJECT}/serviceAccount"
+GCS_SA="... @gs-project-accounts.iam.gserviceaccount.com"
+```
```

Comment:
Q: Is the `...` here indicating that we use some part of the output from the previous `curl` command?

Reply:
Yes. Fixed the text around it to explain what is going on.